### ISSUE TITLE
Load CSVs hosted on same Location (e.g. GitHub pages)

### ISSUE THREAD

**Issue by gustavz** [thoughtworks/build-your-own-radar#111]
What modifications would be necessary to be able to load csv files which are hosted on the same location but behind a proxy like on GitHub Enterprise Pages?d

Example:
- GitHub Enterprise
  - Project Repo
     - docs
       - radar
         - index.html ...
       - sheets
         - examplefile.csv

**Comment by DaniruKun** [comment]
I also agree this is a much needed feature. In our use case, external CSV's and Sheets are on another network and require separate network access. Having the CSVs right in the project would be very helpful.
